docs/quickstart.rst

```diff
@@ -87,6 +97,25 @@ than once to use additional operators.
     $ stac-client search ${STAC_API_URL} -c sentinel-2-l2a --bbox -72.5 40.5 -72 41 --datetime 2020-01-01/2020-01-31 --query "eo:cloud_cover<10" "eo:cloud_cover>5" --matched
     4 items matched
 
+
+Collection searches can also use multiple filters like this example
+search for collections that include the term ``"biomass"`` and have
+a spatial extent that intersects Scandinavia.
+
+.. code-block:: console
+
+    $ stac-client collections https://emc.spacebel.be --q biomass --bbox 0.09 54.72 33.31 71.36  --matched
+    43 items matched
+
+Since most STAC APIs have not yet implemented the collection search 
+extension, ``pystac-client`` will perform a limited client-side 
```

Comment:
```suggestion
Since most STAC APIs have not yet implemented the `collection search 
extension <https://github.com/stac-api-extensions/collection-search>`_, ``pystac-client`` will perform a limited client-side 
```